rtv/content.py

```diff
@@ -384,6 +385,12 @@ def from_name(cls, reddit, name, loader, order=None, query=None):
             else:
                 submissions = reddit.user.get_submitted()
 
+        elif name == 'saved':
+            if not reddit.is_oauth_session():
+                raise exceptions.AccountError('Not logged in')
+            else:
+                submissions = reddit.user.get_saved()
```

Comment:
I see that you've structured this so that users are only allowed to save/view submissions. Does get_saved() only return submissions, or does it return both submissions and comments? If you keep it this way, I think that you will need to add a filter here to return only submission objects.

note: I think that adding support for saved comments would be awesome, but it would require a larger code change. I'll leave it up to you whether you want to try to implement it or not.